package.json

```diff
@@ -41,6 +42,10 @@
         "test:only": "jest test",
         "verify:docs": "sh ci/verify-docs.sh"
     },
+    "dependencies": {
+        "eslint-config-airbnb": "^18.0.1",
+        "eslint-config-prettier": "^6.4.0"
```

Comment:
If we adding it to `dependencies`, do we still need it in `devDependencies`?

Reply:
We don't, removed